dev-python/pyphen/pyphen-0.9.4-r1.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+PYTHON_COMPAT=( python{2_7,3_6,3_7,3_8} )
```

Comment:
do not keep py27

Reply:
Check, no reverse dependencies on python2.7-only apps.